app/models/bot/alegre.rb

```diff
@@ -151,7 +151,7 @@ def self.run(body)
       if body.dig(:event) == 'create_project_media' && !pm.nil?
         Rails.logger.info("[Alegre Bot] [ProjectMedia ##{pm.id}] This item was just created, processing...")
         self.get_language(pm)
-        if self.get_pm_type(pm) == "audio" || self.get_pm_type(pm) == "image"
+        if self.get_pm_type(pm) == "audio" || self.get_pm_type(pm) == "image" || self.get_pm_type(pm) == "video"
```

Comment:
For simplicity you can use this one
`['audio', 'image', 'video'].include?(self.get_pm_type(pm))`